pkg/reconciler/mongodb/mongodbsource_test.go

```diff
@@ -39,8 +46,39 @@ import (
 	. "knative.dev/pkg/reconciler/testing"
 )
 
+var (
+	sinkDNS    = "sink.mynamespace.svc." + utils.GetClusterDomainName()
+	sinkURI, _ = apis.ParseURL("http://" + sinkDNS)
+
+	sinkDestURI = duckv1.Destination{
+		URI: apis.HTTP(sinkDNS),
+	}
+	sinkDest = duckv1.Destination{
+		Ref: &duckv1.KReference{
+			Name:       sinkName,
+			Namespace:  testNS,
+			Kind:       "Channel",
+			APIVersion: "messaging.knative.dev/v1beta1",
+		},
+	}
+)
+
 const (
-	testRAImage = "github.com/googleinterns/knative-source-mongodb/test/image"
+	image          = "image"
+	RAImage        = "github.com/googleinterns/knative-source-mongodb/test/image"
+	sourceName     = "test-MongoDb-source"
+	sourceUID      = "1234"
+	sourceNameLong = "test-MongoDbserver-source-with-a-very-long-name"
+	sourceUIDLong  = "cafed00d-cafed00d-cafed00d-cafed00d-cafed00d"
+	testNS         = "testnamespace"
+	testSchedule   = "*/2 * * * *"
+	testData       = "data"
+	crName         = "knative-eventing-MongoDbsource-adapter"
```

Comment:
do you need this?


I'd suggest you remove all the constants that you are not using... whenever you add a new test that needs them, you add them back

Reply:
Sounds good!